src/business-logic/services/timeout-callback-scheduler.ts

```diff
@@ -1,27 +1,29 @@
 import { CallbackScheduler } from './interfaces/callback-scheduler'
+import { ConsoleLogger } from '../../console-logger'
+import { Logger } from '../../logger'
 
 const EXECUTE_CALLBACK_DURATION_THRESHOLD_IN_MS: number = 1
 const SCHEDULE_RESOLUTION: number = 2
 
 export class TimeoutCallbackScheduler implements CallbackScheduler {
   private static instance: CallbackScheduler
 
-  public static getInstance(): CallbackScheduler {
+  public static getInstance(logger: ConsoleLogger): CallbackScheduler {
     if (!this.instance) {
-      this.instance = new TimeoutCallbackScheduler()
+      this.instance = new TimeoutCallbackScheduler(logger)
     }
     return this.instance
   }
-
+  private readonly logger: Logger
```

Comment:
Mising space between the `getInstance` and the next variable

Reply:
Added new line